flink-scala/src/main/scala/org/apache/flink/api/scala/extensions/acceptPartialFunctions/package.scala

```diff
@@ -0,0 +1,174 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.api.scala.extensions
+
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.api.scala._
+
+import scala.reflect.ClassTag
+
+/**
+  * acceptPartialFunctions extends the original DataSet with methods with unique names
+  * that delegate to core higher-order functions (e.g. `map`) so that we can work around
+  * the fact that overloaded methods taking functions as parameters can't accept partial
+  * functions as well. This enables the possibility to directly apply pattern matching
+  * to decompose inputs such as tuples, case classes and collections.
+  *
+  * e.g.
+  * {{{
+  *   object Main {
+  *     import org.apache.flink.api.scala.extensions._
```

Comment:
Yup, leftover from the previous implementation, thanks for pointing it out, I'll fix this.